config/initializers/traject_initializer.rb

```diff
@@ -1,4 +1,7 @@
-require File.join(Rails.root, 'marc_to_solr', 'lib', 'location_processor_service')
+if Rails.env.test? || Rails.env.development?
+  LocationMapsGeneratorService.generate_from_templates
```

Comment:
Does this mean we should remove the readme instructions to copy the files?

Reply:
I didn't see anything in the README about that, but if there is a mention somewhere of copying the files, we should remove it.

Reply:
@eliotjordan it was added in https://github.com/pulibrary/marc_liberation/pull/1013. maybe you need to rebase?

Reply:
I see it now. I was searching for different terms. Pushed a commit to remove that section.